package.json

```diff
@@ -1,105 +1,43 @@
-      "default": "./dist/router/server.js"
-    }
-  },
-  "bin": {
-    "waku": "./dist/cli.js"
-  },
-  "files": [
-    "src",
-    "dist"
-  ],
+  "private": true,
   "scripts": {
-    "dev": "swc src -d dist -w",
-    "compile": "rm -rf dist && npm run compile:code && npm run compile:types",
-    "compile:code": "swc src -d dist && swc src -d dist/cjs -C module.type=commonjs && echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json",
-    "compile:types": "tsc --project tsconfig.build.json",
+    "dev": "pnpm -r --filter='./packages/*' run dev",
+    "compile": "pnpm -r --filter='./packages/*' run compile",
+    "postinstall": "test -d ./packages/waku/dist || (pnpm -r --filter='./packages/waku' run compile && pnpm link ./packages/waku)",
     "test": "prettier -c . && eslint . && tsc --project . --noEmit",
     "e2e": "playwright test",
-    "examples:dev": "npm run compile:code && (cd ./examples/${NAME} && npm run dev)",
-    "examples:dev:01_counter": "NAME=01_counter npm run examples:dev",
-    "examples:dev:02_async": "NAME=02_async npm run examples:dev",
-    "examples:dev:03_promise": "NAME=03_promise npm run examples:dev",
-    "examples:dev:04_callserver": "NAME=04_callserver npm run examples:dev",
-    "examples:dev:05_mutation": "NAME=05_mutation npm run examples:dev",
-    "examples:dev:06_nesting": "NAME=06_nesting npm run examples:dev",
-    "examples:dev:07_router": "NAME=07_router npm run examples:dev",
-    "examples:dev:08_cookies": "NAME=08_cookies npm run examples:dev",
-    "examples:dev:09_cssmodules": "NAME=09_cssmodules npm run examples:dev",
-    "examples:build": "npm run compile:code && (cd ./examples/${NAME} && npm run build)",
-    "examples:prd": "npm run examples:build && (cd ./examples/${NAME} && npm start)",
-    "examples:prd:01_counter": "NAME=01_counter npm run examples:prd",
-    "examples:prd:02_async": "NAME=02_async npm run examples:prd",
-    "examples:prd:03_promise": "NAME=03_promise npm run examples:prd",
-    "examples:prd:04_callserver": "NAME=04_callserver npm run examples:prd",
-    "examples:prd:05_mutation": "NAME=05_mutation npm run examples:prd",
-    "examples:prd:06_nesting": "NAME=06_nesting npm run examples:prd",
-    "examples:prd:07_router": "NAME=07_router npm run examples:prd",
-    "examples:prd:08_cookies": "NAME=08_cookies npm run examples:prd",
-    "examples:prd:09_cssmodules": "NAME=09_cssmodules npm run examples:prd",
-    "website:dev": "npm run compile:code && ./dist/cli.js dev --config ./website/vite.config.ts",
-    "website:build": "npm run compile:code && ./dist/cli.js build --config ./website/vite.config.ts",
-    "website:vercel": "npm run website:build && cp -Lr ./website/dist/.vercel/output ./.vercel/",
-    "website:prd": "npm run website:build && ./dist/cli.js start --config ./website/vite.config.ts"
-  },
-  "license": "MIT",
-  "engines": {
-    "node": ">=18.0.0"
+    "examples:dev": "(cd ./examples/${NAME} && pnpm run dev)",
```

Comment:
pnpm run X is same as pnpm X

Reply:
Yeah, I actually did so, but reverted it. Just to be explicit about "running" the script.